README.md

```diff
@@ -172,13 +176,11 @@ as long as you have one of the 3 supported keywords right in front
 of the mandatory marker):
 
 ```
-// @todo #224
-/* @todo #TEST-13 */
-# @todo #55:45min
-@todo #67/DES
-;; @todo #678:40m/DEV
-// TODO: #1:30min
-(* TODO #42 *)
```

Comment:
The same: why it has been removed?

Reply:
It is not supported in the code, there is only a logic to remove prefix, but no logic to remove any kinds of symbols from the end